### PR TITLE
Migrated to use towncrier for change logs

### PR DESCRIPTION
**Pull request by andy-maier**
You can display the change log for the upcoming version 1.4.0 by running:
```
make develop
towncrier build --draft
```
with this branch checked out.

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -282,6 +282,103 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
+---------------
+
```
Comment (KSchopmeyer) [inline docs/development.rst:289]: I made a couple of change proposals for pywbem in this area.  They apply here, just to clarify a bith this whole process.

File: changes/1405.cleanup.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Removed code and tests that depends on the differences between Python 3.8+ and
```
Comment (KSchopmeyer) [inline changes/1405.cleanup.rst:1]: depends to depend

File: changes/1436.feature.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Development: Migrated to use towncrier for change logs.
```
Comment (KSchopmeyer) [inline changes/1436.feature.rst:1]: add "See docs/development.rst for details and usage"

**Review by KSchopmeyer — COMMENTED** [review]
Left some editorial comments on the towncrier steps.   

Pretty much same comments in pywbemtools and pywbem.    

Would it be logical to just have DEVELOP.md point to the  documentation documentation instead of having sort of duplicate descriptions???

**Response by Andy:** There is no duplicate content between DEVELOP.md and development.rst. Nevertheless, it makes sense to integrate the content of DEVELOP.md into development.rst. I have created #1448 for that.

File: docs/development.rst (hunk 1 above)
Comment (andy-maier) [reply docs/development.rst:292]: DONE

File: changes/1436.feature.rst (hunk 3 above)
Comment (andy-maier) [reply changes/1436.feature.rst:1]: DONE. Also in pywbem

File: changes/1405.cleanup.rst (hunk 2 above)
Comment (andy-maier) [reply changes/1405.cleanup.rst]: DONE

**Pull request merged**